src/components/Amalgamation/ResultingBusinessName.vue

```diff
@@ -31,7 +31,7 @@
             :businessId="getBusinessId"
             :companyName="companyName"
             :correctionNameChoices="correctionNameChoices"
-            :entityType="getEntityType"
+            :entityType="null"
```

Comment:
This is so that the NR's legal (entity) type is not verified. See https://github.com/bcgov/bcrs-shared-components/blob/aeef17ca8038539fd84d18415e27a5e46fe71d3f/src/components/correct-name/CorrectNameRequest.vue#L181.